### PR TITLE
Feature transpiler mapping

### PR DESCRIPTION
**Pull request by PhBrz**
@DominikBritz It would be great if you could look at the PowerShell script changes.
@dev-sai-collab It would be great if you could look at the transpiler changes.

Thanks!

### CODE REVIEW

File: config-dev/Security inventory/Windows/Certificate/Certificate.ps1 [hunk 1]
```diff
@@ -525,54 +525,64 @@ function Get-vlCertificateCheck {
...
 
-    if ($params.Contains("all") -or $params.Contains("protectedRoots")) {
-        $protectedRoots = Get-vlRootCertificateInstallationCheck    
+    if ($params.Contains("all") -or $params.Contains("CProtRoot")) {
+        $protectedRoots = Get-vlRootCertificateInstallationCheck
         $Output += [PSCustomObject]@{
-            Name         = "Certificate - protectedRoots"
+            Name         = "CProtRoot"
+            DisplayName  = "Protected Root Certificates"
+            Description  = "Checks if the root certificates can be installed by a user."
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/Certificate/Certificate.ps1:533]: delete `the` before root

File: config-dev/Security inventory/Windows/Certificate/Certificate.ps1 [hunk 2]
```diff
@@ -525,54 +525,64 @@ function Get-vlCertificateCheck {
...
 
-    if ($params.Contains("all") -or $params.Contains("protectedRoots")) {
-        $protectedRoots = Get-vlRootCertificateInstallationCheck    
+    if ($params.Contains("all") -or $params.Contains("CProtRoot")) {
+        $protectedRoots = Get-vlRootCertificateInstallationCheck
         $Output += [PSCustomObject]@{
-            Name         = "Certificate - protectedRoots"
+            Name         = "CProtRoot"
+            DisplayName  = "Protected root certificates"
+            Description  = "Checks if root certificates can be installed by a user."
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/Certificate/Certificate.ps1:533]:
```suggestion
            Description  = "Checks if root certificates can be installed by users."
```

File: config-dev/Security inventory/Windows/Certificate/Certificate.ps1 [hunk 3]
```diff
@@ -525,54 +525,64 @@ function Get-vlCertificateCheck {
...
         }
     }
-    if ($params.Contains("all") -or $params.Contains("trustedByWindows")) {
+    if ($params.Contains("all") -or $params.Contains("CTrByWin")) {
         $ctlCheck = Get-vlGetCTLCheck
         $Output += [PSCustomObject]@{
-            Name         = "Certificate - trustedByWindows"
+            Name         = "CTrByWin"
+            DisplayName  = "Certificates trusted by Windows"
+            Description  = "Checks if the certificates are trusted by Windows using the CTL."
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/Certificate/Certificate.ps1:585]:
```suggestion
            Description  = "Checks if the certificates are trusted by Windows using the Certificate Trust List."
```

File: config-dev/Security inventory/Windows/PowerShell/PowerShell.ps1 [hunk 4]
```diff
@@ -610,10 +617,12 @@ function Get-vlPowerShellCheck {
...
     }
 
-    if ($params.Contains("all") -or $params.Contains("powershellversion")) {
+    if ($params.Contains("all") -or $params.Contains("PSVersion")) {
         $powerShellMode = Get-vlPowerShellVersion
         $Output += [PSCustomObject]@{
-            Name         = "powerShellVersion"
+            Name         = "PSVersion"
+            DisplayName  = "PowerShell version"
+            Description  = "Current used PowerShell Version"
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/PowerShell/PowerShell.ps1:625]:
```suggestion
            Description  = "The PowerShell version in use"
```

**Review by DominikBritz — APPROVED** [review]

**Review by dev-sai-collab — APPROVED** [review]
Looks good to me. Nice work.

**Pull request merged**